pagopa/io-web-profile
# [IOPID-638] : Starting with unit tests

## Short description

Some unit tests have been added using Jest for several pages in order to provide adequate coverage.

## How to test

To test, run yarn test to execute the tests, and yarn test --coverage to see the coverage.
